Source/SLKTextViewController.m

```diff
@@ -186,6 +186,11 @@ - (void)viewWillAppear:(BOOL)animated
 {
     [super viewWillAppear:animated];
     
+    // Deactivate automatic scrollView adjustment
```

Comment:
Remove this code because it duplicates the code in the tableView initialization.